src/Model.js

```diff
@@ -184,6 +183,59 @@ const Model = class Model {
     }
 
     /**
+     * Update many-many relations for model.
+     * @param relations
+     */
+    _refreshMany2Many(relations) {
+        const ThisModel = this.getClass();
+        const This = this;
```

Comment:
Nothing changed - had no time to find fix babel/jest settings tests to support it. Right now it can't be changed for that:
```js
_refreshMany2Many = (relations) => {
...
}
```

tests will fail.